src/scenes/AcceleratorRouter/Deliverables/QuestionsDeliverableView.tsx

```diff
@@ -322,7 +327,7 @@ const QuestionBox = ({
               <Grid item xs={12}></Grid>
               <Grid item xs={12}>
                 <DeliverableVariableDetailsInput
-                  values={variable.values}
+                  values={pendingValues || variable.values}
```

Comment:
@nickgraz this change resolves the issue of not being able to type into text fields when editing a Questionnaire Deliverable, but I'm not sure if there's a better approach or anything else to consider here. I tested editing with other field types and it seems like everything works as expected.

Reply:
This makes sense to me. Seems like a strange bug but it makes sense to pass it in like this since this view is the container.